web/src/message_actions_popover.ts

```diff
@@ -27,6 +28,21 @@ let message_actions_popover_keyboard_toggle = false;
 function get_action_menu_menu_items(): JQuery {
     return $("[data-tippy-root] #message-actions-menu-dropdown li:not(.divider):visible a");
 }
+function handleCopyToClipboard(
```

Comment:
This should not be using camelCase, and you should leave a blank line between functions. 

This is also not a good place for this in the file order...